src/interfaces.ts

```diff
@@ -330,18 +330,18 @@ export interface IConversationDetails {
     description?: string;
     roles: IConversationRoles;
     isPublic: boolean;
-    participants?: IConversationParticipant[];
 }
 
 /**
  * Conversation details interface
  */
 export interface IConversationDetails2 extends IConversationDetails {
```

Comment:
Ok so perhaps just a naming issue then, but I guess one that would cause a breaking change to alter now, at least if anyone is consuming the TS interface.